pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[project]
+name = "mcp-server-sql-analyzer"
+version = "0.1.0"
+description = "SQL analyzer server for MCP"
+readme = "README.md"
+requires-python = ">=3.12"
+dependencies = [
+    "mcp>=1.2.0",
+    "sqlglot>=26.2.1",
+    "pydantic>=2.10.6",
+]
+
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/mcp_server_sql_analyzer"]
+
+[project.scripts]
+mcp-server-sql-analyzer = "mcp_server_sql_analyzer:main"
+
+[tool.ruff]
+line-length = 88
+indent-width = 4
+target-version = "py312"
```

Comment:
ruff will respect `project.requires-python`, so you shouldn't need to specify this
```suggestion
```